### PR TITLE
Increase XBE_MAX_VA for XBLA Launcher

### PR DESCRIPTION
**Pull request by LukeUsher**

### CODE REVIEW

**Comment by PatrickvL** [comment]
Where can I find XBLA Launcher?

**Comment by LukeUsher** [comment]
It was a compilation disc of Xbox Live Arcade Games that you had to mail order from Microsoft.

**Comment by LukeUsher** [comment]
If you can source an "Xbox Live Starter kit" it's usually included with that too.

File: src/CxbxKrnl/CxbxKrnl.h [hunk 1]
```diff
@@ -82,7 +82,7 @@ typedef uint32 xbaddr;
 // fit in 51 MB. If we ever encounter an even larger XBE, this
```
Comment (Sarkie) [inline src/CxbxKrnl/CxbxKrnl.h]: change comment now?

Comment (LukeUsher) [inline src/CxbxKrnl/CxbxKrnl.h]: Good spot, feel free to make a pull-request ;)

**Pull request merged**